sh_scrapy/settings.py

```diff
@@ -20,7 +21,7 @@ def _update_settings(o, d):
     # doesn't work if the AWS keys are passed as unicode. Other code may also
     # depend on settings being str. TODO: we should test this
     for k, v in d.items():
-        d[k.encode('utf-8')] = v.encode('utf-8') if isinstance(v, unicode) else v
+        d[to_native_str(k)] = to_native_str(v) if is_string(v) else v
```

Comment:
This is the only place when we have to check a type of `v` before calling `to_native_str`, because there can be integers and probably some other types as well.